vocode/streaming/agent/chat_gpt_agent.py

```diff
@@ -113,6 +121,20 @@ async def generate_response(
             return
         assert self.transcript is not None
 
+        def get_last_user_message():
+            for message in self.transcript.event_logs[::-1]:
+                if message.sender == Sender.HUMAN:
+                    return message.to_string()
+
+        if self.agent_config.vector_db_config:
+            docs_with_scores = await self.vector_db.similarity_search_with_score(
+                get_last_user_message()
+            )
+            self.transcript.add_bot_message(
```

Comment:
should it be added to the transcript as a vector lookup?

i think we should reserve bot messages for actual bot messages. we do a separate type for actions right?

Reply:
Good idea. Implemented this.